src/components/molecules/UserSearchBar/UserSearchBar.tsx

```diff
@@ -55,37 +52,33 @@ const UserSearchBar: FC<UserSearchBarProps> = ({ onSelect }) => {
         <div className="user-search-close-icon" onClick={clearSearchQuery} />
       )}
       {!!numberOfSearchResults && (
-        <>
-          <div className="user-search-results">
-            <div className="user-search-result-number">
-              <b>{numberOfSearchResults}</b> search results
-            </div>
-            {searchResult.map((user, index) => {
-              return (
-                <div
-                  className="row"
-                  key={`room-${index}`}
-                  onClick={() => onSelect(user)}
-                >
-                  <div
-                    className="result-avatar"
-                    style={{
-                      backgroundImage: `url(${user.pictureUrl})`,
-                    }}
-                  ></div>
-                  <div className="result-info">
-                    <div key={`user-${index}`}>{user.partyName}</div>
-                  </div>
-                </div>
-              );
-            })}
+        <div className="user-search-results">
+          <div className="user-search-result-number">
+            <b>{numberOfSearchResults}</b> search results
           </div>
-        </>
+          {searchResult.map((user) => (
```

Comment:
nit: consider renaming to `searchResults`

Reply:
Good catch, thank you!